Kendo-UI-Bootstrap-Integration/src/Kendo-UI-Bootstrap-Integration/Controllers/HomeController.cs

```diff
@@ -6,6 +6,7 @@
 using Kendo_UI_Bootstrap_Integration.Models;
 using Kendo.Mvc.UI;
 using Newtonsoft.Json;
+using KendoUI_Bootstrap_Integration.Models;
```

Comment:
After changing the namespace to match the others used in the app - Kendo_UI_Bootstrap_Integration.Models - remove the "using KendoUI_Bootstrap_Integration.Models;" as the correct one is already present in the controller.